core/web/schema/type/job_proposal.graphql

```diff
@@ -0,0 +1,11 @@
+type JobProposal {
+    id: ID!
+    spec: String!
+    status: String!
```

Comment:
Let's make this an enum

`enum JobProposalStatus`

Reply:
Added